src/react/deckgl.js

```diff
@@ -89,6 +93,8 @@ export default class DeckGL extends React.Component {
 
     // Note: avoid React setState due GL animation loop / setState timing issue
     this.layerManager = new LayerManager({gl});
+    this.effectManager = new EffectManager({gl, deckgl: this});
+    this.effectManager.addEffect(new ReflectionEffect());
```

Comment:
This should be up to the application right? Should we add another `effects` property to the DeckGL component?

<DeckGL layers=[...] effects=[...]/>